cli/src/main/java/com/devonfw/tools/ide/io/AbstractIdeProgressBar.java

```diff
@@ -33,6 +31,15 @@ public long getMaxLength() {
    */
   protected abstract void doStepBy(long stepSize, long currentProgress);
 
+  protected void stepTo(long stepPosition) {
```

Comment:
```suggestion
  /**
   * Sets the progress bar to given step position while making sure to avoid overflow.
   *
   * @param stepPosition position to set to.
   */
  protected void stepTo(long stepPosition) {
```